setup.cfg

```diff
@@ -0,0 +1,52 @@
+[metadata]
+name = interop_tests
+version = 1.0.0
+description = A collection of utility programs used in assisting interop scenario testing.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/rhpit/interop-tests
+author = Cloud Services & Portfolio Integration
+author_email = cspi-qe@redhat.com
+license = GPL-3.0
+license_file = LICENSE
+classifiers =
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    License :: OSI Approved :: GPLv3 License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+
+[options]
+packages = find:
+install_requires =
+    click
+python_requires = >=3.8
```

Comment:
Should we bump this minimal version to be 3.10 as that is what we are defining in the docker file as the base image?